File: connecttoapilibrary/src/main/java/com/sec/connecttoapilibrary/onlinefm/OnLineFMPlayerListener/SubPlayerListener.java
package com.sec.connecttoapilibrary.onlinefm.OnLineFMPlayerListener;

/**
 * Created by SRC-TJ-MM-BinYang on 2018/2/7.
 */

public interface SubPlayerListener {
    void add(ObserverPlayerListener observerListener);
    void notifyObserver(String mPlayer_Url, int play_type);
    void notifyObserverDNS(boolean isStart);
    void remove(ObserverPlayerListener observerListener);
}

File: myonlinefm/src/main/java/com/sec/myonlinefm/OnLineFMPlayerListener/SubPlayerListener.kt
package com.sec.myonlinefm.OnLineFMPlayerListener


/**
 * Created by SRC-TJ-MM-BinYang on 2018/2/7.
 */
interface SubPlayerListener {
    open fun add(observerListener: ObserverPlayerListener?)
    open fun notifyObserver(mPlayer_Url: String?, play_type: Int)
    open fun notifyObserverDNS(isStart: Boolean)
    open fun remove(observerListener: ObserverPlayerListener?)
}